Makefile.core.mk

```diff
@@ -21,6 +21,7 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 3.0.0
 MINOR_VERSION := $(shell v='$(VERSION)'; echo "$${v%.*}")
+MINIMUM_SUPPORTED_VERSION ?= v3.0
```

Comment:
Looks like you added this back when you had a merge conflict with one of my PRs.